partitura/score.py

```diff
@@ -2773,20 +2810,228 @@ class RomanNumeral(TimedObject):
         See parameters
     """
 
-    def __init__(self, text):
+    def __init__(self, text, inversion=None, local_key=None, primary_degree=None, secondary_degree=None, quality=None):
+        super().__init__(text)
+        self.text = text
+        self.accepted_qualities = ('7', 'aug', 'aug6', 'aug7', 'dim', 'dim7', 'hdim7', 'maj', 'maj7', 'min', 'min7')
```

Comment:
Should this list be externalize in the global file or a similar place? Also what is aug7 ?

Reply:
The tests are implemented within the class, so I would say it is better if it stays here, at least for the moment. Of course that is up to discusion